<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Reka Toth" />


<title>MeDeCom results</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="site_libs/anchor-sections-1.0/anchor-sections.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">CoO_Lung_Cancer</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://gitlab.com/tkike/Lung_CoO">
    <span class="fa fa-gitlab"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">MeDeCom results</h1>
<h4 class="author">Reka Toth</h4>
<h4 class="date">19 March, 2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-03-19
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>CoO_Lung_Cancer/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2.9000). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div class="panel-group" id="workflowr-checks">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191010code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191010)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191010code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191010)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
V:/Reka/33_CoO_lung/CoO_Lung_Cancer/data/selected_markers_MCA.txt
</td>
<td style="text-align:left;">
data/selected_markers_MCA.txt
</td>
</tr>
<tr>
<td style="text-align:left;">
V:/Reka/33_CoO_lung/CoO_Lung_Cancer/data/lung_cell_atlas_other_markers.txt
</td>
<td style="text-align:left;">
data/lung_cell_atlas_other_markers.txt
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgitlabcomtkikeLungCoOtree57f1e45862d0ac93af3d8115d8ee13c511ce7451targetblank57f1e45a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://gitlab.com/tkike/Lung_CoO/tree/57f1e45862d0ac93af3d8115d8ee13c511ce7451" target="_blank">57f1e45</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgitlabcomtkikeLungCoOtree57f1e45862d0ac93af3d8115d8ee13c511ce7451targetblank57f1e45a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://gitlab.com/tkike/Lung_CoO/tree/57f1e45862d0ac93af3d8115d8ee13c511ce7451" target="_blank">57f1e45</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.Rhistory
    Ignored:    data/DML_results_MsCCSP_MsSPC_comparison.RDS
    Ignored:    data/DML_results_MsCCSP_MsSPC_control_comparison.RDS
    Ignored:    data/DML_results_MsCCSP_tumor_control_comparison.RDS
    Ignored:    data/DML_results_MsCCSP_tumor_rfp_control_comparison.RDS
    Ignored:    data/DML_results_MsCCSP_tumor_rfp_tumor_comparison.RDS
    Ignored:    data/DML_results_MsSPC_tumor_control_comparison.RDS
    Ignored:    data/DML_results_MsSPC_tumor_rfp_control_comparison.RDS
    Ignored:    data/DML_results_MsSPC_tumor_rfp_tumor_comparison.RDS
    Ignored:    data/DMLs_noMsCCSP_control01_smoothed.RDS
    Ignored:    data/DMRs_noMsCCSP_control01_smoothed.RDS
    Ignored:    data/QC_data.RData
    Ignored:    data/annotation_table_TAGWGBS_02_10.txt
    Ignored:    data/genotypes_all.txt
    Ignored:    data/methrix_object.RDS
    Ignored:    data/no_snps_methrix.RDS
    Ignored:    data/promoter_beds/
    Ignored:    data/snp_data_5_strains_14_10.RDS
    Ignored:    data/snp_ranges.bed
    Ignored:    data/snp_ranges.bedGraph
    Ignored:    docs/01_read_in_MethylDackel.html
    Ignored:    docs/02-DMR_calling.html
    Ignored:    docs/DMR_report.html
    Ignored:    docs/PCA_plots.html
    Ignored:    docs/QC_report.html
    Ignored:    docs/about.html
    Ignored:    docs/index.html
    Ignored:    docs/license.html
    Ignored:    docs/methrix_reports.html
    Ignored:    docs/site_libs/jqueryui-1.11.4/index.html
    Ignored:    enrichment_new_groups/

Untracked files:
    Untracked:  analysis/02-DMR_calling.Rmd
    Untracked:  analysis/02-DMR_calling_new_groups.Rmd
    Untracked:  analysis/DMR_enrichment.Rmd
    Untracked:  analysis/DMR_report.Rmd
    Untracked:  analysis/DMR_report_new_groups.Rmd
    Untracked:  analysis/DMR_report_selected.Rmd
    Untracked:  analysis/Figures.Rmd
    Untracked:  analysis/LMC_annotation.Rmd
    Untracked:  analysis/LMC_based_enhancer_enrichment.R
    Untracked:  analysis/LMC_promoters.Rmd
    Untracked:  analysis/MeDeCom.Rmd
    Untracked:  analysis/MeDeCom_MsCCSP.Rmd
    Untracked:  analysis/MeDeCom_all.Rmd
    Untracked:  analysis/MeDeCom_all_no_MsCCSP_c4.Rmd
    Untracked:  analysis/MeDeCom_controls.Rmd
    Untracked:  analysis/MeDeCom_controls_no_MsCCSP_c4.Rmd
    Untracked:  analysis/MeDeCom_poised_enhancers.Rmd
    Untracked:  analysis/PCA_plots.Rmd
    Untracked:  analysis/PCA_plots_selected.Rmd
    Untracked:  analysis/QC_report.Rmd
    Untracked:  analysis/cNMF.Rmd
    Untracked:  analysis/candiate_region_plot.Rmd
    Untracked:  analysis/candiate_region_plot_histogram.Rmd
    Untracked:  analysis/candiate_region_plot_histogram_new_groups.Rmd
    Untracked:  analysis/candidate_selection.Rmd
    Untracked:  analysis/combined_candidate_plots.Rmd
    Untracked:  analysis/combined_differential.Rmd
    Untracked:  analysis/combined_enhancer_plots.Rmd
    Untracked:  analysis/dmr_calling_methrix_17_07_noMsCCSP_control01_smoothed.R
    Untracked:  analysis/enhancers.rmd
    Untracked:  analysis/go_region_plot_histogram.Rmd
    Untracked:  analysis/homer_all.Rmd
    Untracked:  analysis/homer_enhancers.Rmd
    Untracked:  analysis/homer_new_groups.Rmd
    Untracked:  analysis/prepare_homer.Rmd
    Untracked:  analysis/prepare_homer_alldmr.Rmd
    Untracked:  analysis/prepare_homer_alldmr_new.Rmd
    Untracked:  analysis/prepare_medecom_regions.Rmd
    Untracked:  analysis/prepare_promoters_tca.Rmd
    Untracked:  analysis/tcga_medecom.Rmd
    Untracked:  code/EpiScore.R
    Untracked:  code/Figure_1.R
    Untracked:  code/Figure_2.R
    Untracked:  code/Figure_3.R
    Untracked:  code/LMC_correlation_enrichment.R
    Untracked:  code/LMC_enrichment.R
    Untracked:  code/color_palette.R
    Untracked:  code/gviz_region_plot_figures.R
    Untracked:  code/medecom_poised_enhancers_all.R
    Untracked:  code/medecom_poised_enhancers_all_hopx.R
    Untracked:  code/medecom_poised_enhancers_all_hopx_no1.R
    Untracked:  code/medecom_poised_enhancers_all_norfp.R
    Untracked:  code/medecom_poised_enhancers_all_smoothed.R
    Untracked:  code/presentation_heatmap.R
    Untracked:  code/shiny_app/
    Untracked:  code/tcga_medecom_corr.R
    Untracked:  code/tcga_medecom_correlation.RData
    Untracked:  code/tcga_shiny.R
    Untracked:  data/.gitignore
    Untracked:  data/AU040972cand.pdf
    Untracked:  data/CD74cand.pdf
    Untracked:  data/Cyp2f2cand.pdf
    Untracked:  data/DML_results_MsCCSP_MsSPC_tumor_comparison.RDS
    Untracked:  data/DML_results_new_group_MsCCSP_MsSPC_control_comparison.RDS
    Untracked:  data/DML_results_new_group_MsCCSP_MsSPC_tumor_comparison.RDS
    Untracked:  data/DML_results_new_group_MsCCSP_tumor_control_comparison.RDS
    Untracked:  data/DML_results_new_group_MsSPC_tumor_control_comparison.RDS
    Untracked:  data/DMLs_new_groups.RDS
    Untracked:  data/DMRs_new_groups.RDS
    Untracked:  data/E-ENAD-15-marker-genes-files_Mouse_cell_atlas.zip
    Untracked:  data/E-ENAD-15.marker_genes_inferred_cell_type_-_ontology_labels.tsv
    Untracked:  data/ENCODE_SCREEN_lung_postnatal_0_CRE.7group.bed
    Untracked:  data/ENdb_target_gene.txt
    Untracked:  data/FactorViz_outputs.zip
    Untracked:  data/FactorViz_outputs/
    Untracked:  data/Frem1cand.pdf
    Untracked:  data/Fukazawa_2016_Sox2_targets.xlsx
    Untracked:  data/Itih4cand.pdf
    Untracked:  data/Itih4cand_data.RDS
    Untracked:  data/JASPAR_2020_matrix_clustering_vertebrates_archive.zip
    Untracked:  data/Krt19cand.pdf
    Untracked:  data/MeDeCom_MCCSP_SPC_prepared_filt.rds
    Untracked:  data/MsCCPS_MsSPC_enhancers.rds
    Untracked:  data/MsCCPS_MsSPC_promoters.rds
    Untracked:  data/MsCCSP_MsSPC_tumor_different_genes.txt
    Untracked:  data/MsCCSP_tumor_MsCCSP_normal_different_genes.txt
    Untracked:  data/MsSPC_normal_MsCCSP_normal_different_genes.txt
    Untracked:  data/MsSPC_tumor_MsCCSP_tumor_different_genes.txt
    Untracked:  data/MsSPC_tumor_MsSPC_normal_different_genes.txt
    Untracked:  data/Nkx2-1cand.pdf
    Untracked:  data/Nr2f2cand.pdf
    Untracked:  data/Scgb1a1cand.pdf
    Untracked:  data/Scgb1a1cand_data.RDS
    Untracked:  data/Scgb3a2cand.pdf
    Untracked:  data/Scgb3a2cand_data.RDS
    Untracked:  data/Sftpccand.pdf
    Untracked:  data/Sftpccand_data.RDS
    Untracked:  data/Sox2_target.tsv
    Untracked:  data/Sox2cand.pdf
    Untracked:  data/Sox2cand_data.RDS
    Untracked:  data/all_Ts.RDS
    Untracked:  data/all_metascape report.zip
    Untracked:  data/all_samples_enhancers.rds
    Untracked:  data/all_samples_promoters.rds
    Untracked:  data/annotated_regions/
    Untracked:  data/annotation_table_TAGWGBS_16_2_2021.txt
    Untracked:  data/annotation_table_TAGWGBS_17_2.txt
    Untracked:  data/annotation_table_TAGWGBS_24_2.txt
    Untracked:  data/annotation_table_TAGWGBS_31_1.txt
    Untracked:  data/bedgraphs/
    Untracked:  data/beds/
    Untracked:  data/beds_new_groups/
    Untracked:  data/bivalent_enhancers_plot.pdf
    Untracked:  data/candidate_genes.txt
    Untracked:  data/candidate_promoter_methylation.RDS
    Untracked:  data/candidate_promoter_methylation_smoothed.RDS
    Untracked:  data/candidate_promoter_methylation_smoothed_methrix.RDS
    Untracked:  data/candidate_promoter_smoothed_bsseq.RDS
    Untracked:  data/cell_type_markers.gmt
    Untracked:  data/cellphonedb_example_data.zip
    Untracked:  data/comparisons.RDS
    Untracked:  data/dmrs/
    Untracked:  data/enc_epd_enhc_gene_mm10.txt
    Untracked:  data/enc_epd_enhc_gene_mm10.txt.gz
    Untracked:  data/enc_epd_enhc_gene_mm10_int.txt
    Untracked:  data/enhancer_beds/
    Untracked:  data/enhancers/
    Untracked:  data/enhancers_CCSP_genes.txt
    Untracked:  data/enhancers_LMC1_genes.txt
    Untracked:  data/enhancers_LMC2_genes.txt
    Untracked:  data/enhancers_LMC3_genes.txt
    Untracked:  data/enhancers_LMC4_genes.txt
    Untracked:  data/enhancers_genes.RDS
    Untracked:  data/enhancers_genes.txt
    Untracked:  data/enhancers_genes.xlsx
    Untracked:  data/enhancers_genes_nobg.txt
    Untracked:  data/enhancers_genes_nobg.xlsx
    Untracked:  data/figures/
    Untracked:  data/ge_dmr_candidates.rds
    Untracked:  data/homer_res/
    Untracked:  data/homer_res_new_groups/
    Untracked:  data/human_lung_cell_atlas_all_markers.txt
    Untracked:  data/interactive_trees/
    Untracked:  data/labels.RDS
    Untracked:  data/linked_promoter_TSS.xlsx
    Untracked:  data/lung_cell_atlas_other_markers.txt
    Untracked:  data/medecom/
    Untracked:  data/medecom_MsCCSP.RDS
    Untracked:  data/medecom_MsCCSP_MsSPC_enhancers.RDS
    Untracked:  data/medecom_MsCCSP_MsSPC_promoters.RDS
    Untracked:  data/medecom_MsCCSP_SPC_promoters.RDS
    Untracked:  data/medecom_MsCCSP_promoters.RDS
    Untracked:  data/medecom_all.RDS
    Untracked:  data/medecom_all_no_MsCCSP_c4.RDS
    Untracked:  data/medecom_controls.RDS
    Untracked:  data/medecom_proportions_all.RDS
    Untracked:  data/medecom_proportions_all_HOPX.RDS
    Untracked:  data/medecom_random_sites.RDS
    Untracked:  data/medecom_random_sites_all.RDS
    Untracked:  data/metascape_result.tju3krgk_.xlsx
    Untracked:  data/mm9ToMm10.over.chain
    Untracked:  data/mm9ToMm10.over.chain.gz
    Untracked:  data/most_variable_in_controls.rds
    Untracked:  data/new_tumor_groups.RDS
    Untracked:  data/new_tumor_groups_HOPX.RDS
    Untracked:  data/nkx2_1_enrichment/
    Untracked:  data/no_snpsXY_methrix.RDS
    Untracked:  data/no_snps_methrix.qs
    Untracked:  data/promoter_bsseq_smoothed.RDS
    Untracked:  data/promoter_methylation.RDS
    Untracked:  data/promoter_methylation_smoothed.RDS
    Untracked:  data/promoter_methylation_smoothed_methrix.RDS
    Untracked:  data/random_sites.rds
    Untracked:  data/random_sites_all.rds
    Untracked:  data/removed_sites_MeDeCom.RDS
    Untracked:  data/removed_sites_MsCCSP_MeDeCom.RDS
    Untracked:  data/removed_sites_all_MeDeCom.RDS
    Untracked:  data/removed_sites_control_MeDeCom.RDS
    Untracked:  data/selected_markers_MCA.txt
    Untracked:  data/selected_sites_MeDeCom.RDS
    Untracked:  data/selected_sites_MsCCSP_MeDeCom.RDS
    Untracked:  data/selected_sites_all_MeDeCom.RDS
    Untracked:  data/selected_sites_control_MeDeCom.RDS
    Untracked:  data/sequencing_annotation_table.csv
    Untracked:  data/target_genes_nkx2-1.txt
    Untracked:  data/target_genes_nkx2-1.xlsx
    Untracked:  data/tcga_laml_ge.RDS
    Untracked:  data/tcga_luad_ge.RDS
    Untracked:  data/tcga_medecom_corr_LMC1.tiff
    Untracked:  data/tcga_medecom_corr_LMC2.tiff
    Untracked:  data/tcga_medecom_corr_LMC3.tiff
    Untracked:  data/tcga_medecom_corr_LMC4.tiff
    Untracked:  data/tcga_medecom_corr_LMC5.tiff
    Untracked:  data/tcga_medecom_corr_LMC6.tiff
    Untracked:  data/tcga_medecom_corr_LMC7.tiff
    Untracked:  data/tcga_medecom_correlation.RData
    Untracked:  data/~$Fukazawa_2016_Sox2_targets.xlsx
    Untracked:  data/~$linked_promoter_TSS.xlsx
    Untracked:  desktop.ini
    Untracked:  docs/.gitignore
    Untracked:  docs/DMR_enrichment.html
    Untracked:  docs/DMR_report_selected.html
    Untracked:  docs/Figure_1/
    Untracked:  docs/Figure_2/
    Untracked:  docs/Figure_3/
    Untracked:  docs/Figures.html
    Untracked:  docs/PCA_plots_selected.html
    Untracked:  docs/QC_report/
    Untracked:  docs/QC_report_no_SNPs/
    Untracked:  docs/candiate_region_plot.html
    Untracked:  docs/candiate_region_plot_histogram.html
    Untracked:  docs/figure/DMR_enrichment.Rmd/
    Untracked:  docs/figure/DMR_report.Rmd/
    Untracked:  docs/figure/DMR_report_new_groups.Rmd/
    Untracked:  docs/figure/DMR_report_selected.Rmd/
    Untracked:  docs/figure/Figures.Rmd/
    Untracked:  docs/figure/MeDeCom_all.Rmd/
    Untracked:  docs/figure/MeDeCom_all_no_MsCCSP_c4.Rmd/
    Untracked:  docs/figure/MeDeCom_controls.Rmd/
    Untracked:  docs/figure/PCA_plots.Rmd/
    Untracked:  docs/figure/PCA_plots_selected.Rmd/
    Untracked:  docs/figure/prepare_medecom_regions.Rmd/
    Untracked:  docs/medecom_CCSP_controls_plot.RDS
    Untracked:  docs/plots_for_figures/
    Untracked:  docs/prepare_homer.html
    Untracked:  docs/prepare_medecom_regions.html
    Untracked:  docs/site_libs/crosstalk-1.1.0.1/
    Untracked:  output/DML_results_new_groups.RDS
    Untracked:  output/QC_report.html
    Untracked:  output/QC_report/MC_per_chr.tsv
    Untracked:  output/QC_report/MsCCSP_control01_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control02_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control03_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control04_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control05_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control06_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_control07_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor01_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor02.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor02_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor03.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor03_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor04.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsCCSP_tumor04_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_control01_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_control02_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_control03_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_control04_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_control05_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_tumor01_density.tsv.gz
    Untracked:  output/QC_report/MsHOPX_tumor02_density.tsv.gz
    Untracked:  output/QC_report/MsKrt5_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control01_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control02_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control03_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control04_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control05_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_control06_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor01_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor02.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor02_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor03.rfp_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor03_density.tsv.gz
    Untracked:  output/QC_report/MsSPC_tumor04_density.tsv.gz
    Untracked:  output/QC_report/contig_lens.tsv
    Untracked:  output/QC_report/global_MC_per_samp.tsv
    Untracked:  output/QC_report/n_covered_by_all_samples.tsv
    Untracked:  output/QC_report/n_covered_per_chr.tsv
    Untracked:  output/QC_report_deprecated/
    Untracked:  output/QC_report_no_SNPs/MC_per_chr.tsv
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control03_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control04_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control05_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control06_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_control07_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor02.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor03.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor03_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor04.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsCCSP_tumor04_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_control01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_control02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_control03_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_control04_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_control05_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_tumor01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsHOPX_tumor02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsKrt5_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control03_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control04_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control05_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_control06_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor01.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor01_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor02.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor02_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor03.rfp_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor03_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/MsSPC_tumor04_density.tsv.gz
    Untracked:  output/QC_report_no_SNPs/contig_lens.tsv
    Untracked:  output/QC_report_no_SNPs/global_MC_per_samp.tsv
    Untracked:  output/QC_report_no_SNPs/n_covered_by_all_samples.tsv
    Untracked:  output/QC_report_no_SNPs/n_covered_per_chr.tsv
    Untracked:  output/QC_report_no_SNPs_deprecated/
    Untracked:  output/annotated_regions/
    Untracked:  output/candidate_regions_hist/
    Untracked:  output/candidate_regions_point/
    Untracked:  output/enrichment_res/
    Untracked:  output/homer/
    Untracked:  output/poised_enhancers_all_heatmap.pdf
    Untracked:  output/poised_enhancers_controls_heatmap.pdf
    Untracked:  output/poised_enhancers_heatmap.pdf
    Untracked:  output/tca_res.RDS
    Untracked:  output/tca_res_candidate_promoters.RDS
    Untracked:  output/tca_res_promoter_candidate.RDS

Unstaged changes:
    Modified:   analysis/_site.yml
    Modified:   analysis/index.Rmd
    Modified:   code/MeDeCom_poised_enhancers.R
    Modified:   code/gviz_region_plot.R
    Modified:   code/homer_analysis.sh
    Modified:   output/QC_report/methrix_reports.html
    Modified:   output/QC_report_no_SNPs/methrix_reports.html

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/MeDeCom_results_noHopx.Rmd</code>) and HTML (<code>public/MeDeCom_results_noHopx.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://gitlab.com/tkike/Lung_CoO/blob/57f1e45862d0ac93af3d8115d8ee13c511ce7451/analysis/MeDeCom_results_noHopx.Rmd" target="_blank">57f1e45</a>
</td>
<td>
tkike
</td>
<td>
2021-03-19
</td>
<td>
wflow_publish(c(“analysis/MeDeCom_results_noHopx.Rmd”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://glcdn.githack.com/tkike/Lung_CoO/raw/4d58f625f2b6f337797dea498785a7a6e0995243/public/MeDeCom_results_noHopx.html" target="_blank">4d58f62</a>
</td>
<td>
tkike
</td>
<td>
2021-03-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://gitlab.com/tkike/Lung_CoO/blob/229655acc2efd7b612e220a8facc1124cc4d637e/analysis/MeDeCom_results_noHopx.Rmd" target="_blank">229655a</a>
</td>
<td>
tkike
</td>
<td>
2021-03-16
</td>
<td>
wflow_publish(“analysis/MeDeCom_results_noHopx.Rmd”)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>res &lt;- readRDS(paste0(DATA, &quot;no_snps_methrix.RDS&quot;))


##remove MsCCSP_control01, because of bad QC values

res &lt;- methrix::subset_methrix(res, samples = attr(res@colData, &quot;rownames&quot;)[-which(attr(res@colData, &quot;listData&quot;)$full_name==&quot;MsCCSP_control01&quot;)])</code></pre>
<pre><code>Subsetting by samples</code></pre>
<pre class="r"><code>res &lt;- methrix::subset_methrix(res, samples = attr(res@colData, &quot;rownames&quot;)[-which(attr(res@colData, &quot;listData&quot;)$full_name==&quot;MsCCSP_control05&quot;)])</code></pre>
<pre><code>Subsetting by samples</code></pre>
<pre class="r"><code>res &lt;- methrix::subset_methrix(res, samples = attr(res@colData, &quot;rownames&quot;)[-which(attr(res@colData, &quot;listData&quot;)$full_name==&quot;MsKrt5_tumor01-rfp&quot;)])</code></pre>
<pre><code>Subsetting by samples</code></pre>
<pre class="r"><code>#annotation &lt;- read.delim(file=file.path(DATA, &quot;annotation_table_TAGWGBS_02_10.txt&quot;),stringsAsFactors = F)

res &lt;- methrix::subset_methrix(res, contigs = c(paste0(&quot;chr&quot;, 1:19)))</code></pre>
<pre><code>-Subsetting by contigs</code></pre>
<pre class="r"><code>res &lt;- methrix::subset_methrix(res, samples = attr(res@colData, &quot;rownames&quot;)[-which(attr(res@colData, &quot;listData&quot;)$full_name==&quot;MsHOPX_control01&quot;)])</code></pre>
<pre><code>Subsetting by samples</code></pre>
<pre class="r"><code>mat  &lt;- as.data.frame(readRDS(paste0(DATA, &quot;medecom/bivalent_enhancers.RDS&quot;)))
rowsds &lt;- rowSds(as.matrix(mat[,-(1:3)]), na.rm=T)
mat &lt;- mat[order(rowsds, decreasing = T)[1:100000],]
mat &lt;- mat[complete.cases(mat),]
#try without Hopx1


mat &lt;- mat[,colnames(mat)!=&quot;MsHOPX_control01&quot;]</code></pre>
<div id="medecom-analysis-using-bivalent-enhancers-in-all-samples---no-hopx-control1" class="section level1">
<h1>MeDeCom analysis using bivalent enhancers in all samples - no Hopx-control1</h1>
<pre class="r"><code># MeDeCom analysis using bivalent enhancers in controls

#medecom.result2 &lt;- MeDeCom::runMeDeCom(as.matrix(mat[,-(1:3)]), 2:6, c(0, 10^(-5:-1)), NINIT = 10, NFOLDS = 10, ITERMAX = 300, NCORES = 8)
#saveRDS(medecom.result2, file = paste0(DATA, &quot;medecom/poised_enhancers.RDS&quot;))

medecom.result2 &lt;-  readRDS( file = paste0(DATA, &quot;medecom/poised_enhancers_new_samples_no_hopx1.RDS&quot;))
MeDeCom::plotParameters(MeDeComSet = medecom.result2)</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Model optimization</p>
<pre class="r"><code>K_sel &lt;- 4
lambda_sel &lt;- 0.001
proportions &lt;- MeDeCom::getProportions(medecom.result2, K=K_sel, lambda=lambda_sel)
colnames(proportions) &lt;- colnames(mat[,-(1:3)])

LMCs &lt;- MeDeCom::getLMCs(medecom.result2, K=K_sel, lambda=lambda_sel)
colnames(LMCs) &lt;- paste0(&quot;LMC&quot;, 1:K_sel)
#LMCs &lt;- cbind(mat[,1:3], LMCs)</code></pre>
<p>Annotation and plotting</p>
<pre class="r"><code>annotation &lt;- data.frame(staining=gsub(&quot;(Ms[[:alnum:]]+)_(tumor|control)[[:digit:]]+(.rfp)?&quot;, &quot;\\1&quot;, colnames(mat[,-(1:3)])),
                         type= gsub(&quot;(Ms[[:alnum:]]+)_(tumor|control)[[:digit:]]+(.rfp)?&quot;, &quot;\\2&quot;, colnames(mat[,-(1:3)])),
                         origin = ifelse(grepl(&quot;rfp&quot;, colnames(mat[,-(1:3)])), &quot;RFP&quot;, &quot;GFP&quot;))
rownames(annotation) &lt;- colnames(mat[,-(1:3)])
print(xtable::xtable(annotation), type=&quot;html&quot;)</code></pre>
<!-- html table generated in R 4.0.3 by xtable 1.8-4 package -->
<!-- Fri Mar 19 16:04:31 2021 -->
<table border="1">
<tr>
<th>
</th>
<th>
staining
</th>
<th>
type
</th>
<th>
origin
</th>
</tr>
<tr>
<td align="right">
MsCCSP_control02
</td>
<td>
MsCCSP
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_control03
</td>
<td>
MsCCSP
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_control04
</td>
<td>
MsCCSP
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_control06
</td>
<td>
MsCCSP
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_control07
</td>
<td>
MsCCSP
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor01
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor02.rfp
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor02
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor03.rfp
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor03
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor04.rfp
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsCCSP_tumor04
</td>
<td>
MsCCSP
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_control02
</td>
<td>
MsHOPX
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_control03
</td>
<td>
MsHOPX
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_control04
</td>
<td>
MsHOPX
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_control05
</td>
<td>
MsHOPX
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_tumor01.rfp
</td>
<td>
MsHOPX
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_tumor01
</td>
<td>
MsHOPX
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsHOPX_tumor02
</td>
<td>
MsHOPX
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control01
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control02
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control03
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control04
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control05
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_control06
</td>
<td>
MsSPC
</td>
<td>
control
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor01.rfp
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor01
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor02.rfp
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor02
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor03.rfp
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
RFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor03
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
<tr>
<td align="right">
MsSPC_tumor04
</td>
<td>
MsSPC
</td>
<td>
tumor
</td>
<td>
GFP
</td>
</tr>
</table>
<pre class="r"><code>try(pheatmap(proportions,  show_colnames = T, legend = F, annotation_legend = T, fontsize = 12, annotation_col = annotation))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="calculating-correlation" class="section level1">
<h1>Calculating correlation</h1>
<pre class="r"><code>promoters &lt;- build_annotations(&quot;mm10&quot;, &quot;mm10_genes_promoters&quot;)</code></pre>
<pre><code>Loading required package: GenomicFeatures</code></pre>
<pre><code>&#39;select()&#39; returned 1:1 mapping between keys and columns</code></pre>
<pre><code>Building promoters...</code></pre>
<pre class="r"><code>promoter_methylation &lt;- methrix::get_region_summary(res, promoters)</code></pre>
<pre><code>-Checking for overlaps..</code></pre>
<pre><code>-Summarizing by average</code></pre>
<pre><code>-Done! Finished in:35.1s elapsed (34.8s cpu)</code></pre>
<pre class="r"><code>promoter_methylation$symbol &lt;- promoters$symbol[promoter_methylation$rid]
promoter_methylation &lt;- as.data.frame(promoter_methylation)

promoter_methylation &lt;- promoter_methylation %&gt;% 
   filter(n_overlap_CpGs&gt;1) %&gt;% 
   filter(apply(., 1, function(x) sum(is.na(x)))&lt;10) 

promoter_methylation_control &lt;- promoter_methylation %&gt;% 
  select(-grep(&quot;tumor&quot;, colnames(.)))

promoter_methylation &lt;- promoter_methylation[complete.cases(promoter_methylation[,grep(&quot;Ms&quot;, colnames(promoter_methylation))]),]
promoter_methylation_control &lt;- promoter_methylation_control[complete.cases(promoter_methylation_control[,grep(&quot;Ms&quot;, colnames(promoter_methylation_control))]),]



proportions_control &lt;- proportions[,-grep(&quot;tumor&quot;, colnames(proportions))]


datasets_combined &lt;- list(&quot;all&quot;=list(promoter_methylation, proportions), 
                          &quot;control&quot;=list(promoter_methylation_control, proportions_control))

datasets &lt;- datasets_combined[[2]]</code></pre>
<pre class="r"><code>T_all &lt;- MeDeCom:::getT.gini(D=as.matrix(datasets[[1]][,grep(&quot;Ms&quot;, colnames(datasets[[1]]))]), A = datasets[[2]], lambda = lambda_sel)
colnames(T_all) &lt;- paste0(&quot;gini_LMC&quot;, 1:K_sel)
datasets[[1]] &lt;- cbind(datasets[[1]], T_all)

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  &quot;[&lt;-&quot;(paste0(&quot;LMC&quot;, 1:4), value = NA_real_) %&gt;%
   &quot;[&lt;-&quot;(paste0(&quot;LMC&quot;, 1:4, &quot;_p&quot;), value = NA_real_)


for (LMCs in paste0(&quot;LMC&quot;, 1:4)){
  
est &lt;-lapply(1:nrow(datasets[[1]]), function(x)
  cor.test(datasets[[2]][LMCs,],  as.numeric(datasets[[1]][x,-c(1:5, which(colnames(datasets[[1]])==&quot;symbol&quot;), grep(&quot;LMC&quot;, colnames(datasets[[1]])))]), method = &quot;pears&quot;))
datasets[[1]][,LMCs] &lt;- unlist(lapply(est, function(x) x$estimate))
datasets[[1]][,paste0(LMCs, &quot;_p&quot;)] &lt;- -log10(unlist(lapply(est, function(x) x$p.value)))

}

#fun_filt &lt;- function(val, pval){
#  ifelse(abs(val)&gt;0.8 &amp; -log10(pval)&gt;5, symbol, NA)
#}

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(across(c(Sepal.Length, Sepal.Width), round))
    mutate(new_labels=replace(get(label), duplicated(get(label)), NA))
    
datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC1_label=ifelse(abs(LMC1)&gt;0.9 &amp; LMC1_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC2_label=ifelse(abs(LMC2)&gt;0.9 &amp; LMC2_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC3_label=ifelse(abs(LMC3)&gt;0.9 &amp; LMC3_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC4_label=ifelse(abs(LMC4)&gt;0.9 &amp; LMC4_p&gt;5, symbol, NA))


vars &lt;- names(marker_lists_short)


datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(across(colnames(.) %in% vars) ~ case_when(symbol %in% marker_lists_short[[.]]))
  mutate(AT1_markers=ifelse(symbol %in% marker_lists_short$`AT1 cells`, symbol, NA)) %&gt;% 
  mutate(AT2_markers=ifelse(symbol %in% marker_lists_short$`AT2 cells`, symbol, NA)) %&gt;%
  mutate(basal_markers=ifelse(symbol %in% marker_lists_short$`basal cells`, symbol, NA)) %&gt;%
  mutate(ciliated_markers=ifelse(symbol %in% marker_lists_short$`ciliated cells`, symbol, NA)) %&gt;%
  mutate(club_markers=ifelse(symbol %in% marker_lists_short$`club cells`, symbol, NA)) %&gt;%
  mutate(goblet_markers=ifelse(symbol %in% marker_lists_short$`goblet cells`, symbol, NA)) %&gt;%
  mutate(Krt8_markers=ifelse(symbol %in% marker_lists_short$`Krt8 ADI`, symbol, NA)) %&gt;%
  mutate(AT1_AT2_markers=ifelse(symbol %in% marker_lists_short$`Mixed AT1_AT2`, symbol, NA))# %&gt;%
  mutate(club_progenitor_markers=ifelse(symbol %in% marker_lists_short$`Club progenitor`, symbol, NA))


volcano_plot &lt;- ggplot(datasets[[1]], aes(x=LMC1, y=-log10(LMC1_p), label=club_markers)) + 
  geom_point()+geom_label_repel(max.overlaps = Inf)

saveRDS(datasets[[1]], file=file.path(DATA, &quot;medecom/control_correlation_LMCs_no_Hopx.Rmd&quot;))</code></pre>
</div>
<div id="control-samples" class="section level1">
<h1>Control samples</h1>
<div id="correlation-of-promoter-methylation-with-lmcs" class="section level2 tabset tabset-fade tabset-pills">
<h2>Correlation of promoter methylation with LMCs</h2>
<pre class="r"><code>datasets[[1]] &lt;- readRDS( file=file.path(DATA, &quot;medecom/control_correlation_LMCs_no_Hopx.Rmd&quot;))


for (lmcs in paste0(&quot;LMC&quot;, 1:4)){
g &lt;- volcano_plot_correlation(data = datasets[[1]], x = lmcs, y = paste0(lmcs, &quot;_p&quot;), label = paste0(lmcs, &quot;_label&quot;))
print(g)
}</code></pre>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129374 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129306 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129359 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129298 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>for (marks in colnames(datasets[[1]])[grep(&quot;_markers&quot;,colnames(datasets[[1]]) )]){
  
   cat(&#39;\n&#39;)
  
  cat(&quot;### Markers for &quot;,
      gsub(&quot;_markers&quot;, &quot;&quot;, marks), &quot; cells. &quot;,
      &quot;\n&quot;)
  
  for (lmcs in paste0(&quot;LMC&quot;, 1:4)){
g &lt;- volcano_plot_correlation(data = datasets[[1]], x = lmcs, y = paste0(lmcs, &quot;_p&quot;), label = marks)
print(g)
  }
  

       cat(&#39;\n&#39;)
       
 plot_data &lt;- datasets[[1]][!is.na(datasets[[1]][,marks]),]
  plot_data &lt;- plot_data[,grep(&quot;control&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))

  cat(&#39;\n&#39;)
  
       
}</code></pre>
<div id="markers-for-at1-cells." class="section level3">
<h3>Markers for AT1 cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129348 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129348 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129348 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129348 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-8.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-9.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-at2-cells." class="section level3">
<h3>Markers for AT2 cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-13.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-14.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-basal-cells." class="section level3">
<h3>Markers for basal cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129346 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-15.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129346 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-16.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129346 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-17.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129346 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-18.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-19.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-ciliated-cells." class="section level3">
<h3>Markers for ciliated cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129352 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-20.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129352 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-21.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129352 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-22.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129352 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-23.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-24.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-club-cells." class="section level3">
<h3>Markers for club cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-25.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-26.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-27.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129351 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-28.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-29.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-goblet-cells." class="section level3">
<h3>Markers for goblet cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-30.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-31.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-32.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-33.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-34.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-krt8-cells." class="section level3">
<h3>Markers for Krt8 cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-35.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-36.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-37.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129347 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-38.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-39.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-at1_at2-cells." class="section level3">
<h3>Markers for AT1_AT2 cells.</h3>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).</code></pre>
<pre><code>Warning: Removed 129366 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-40.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129366 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-41.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129366 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-42.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 1 rows containing missing values (geom_point).

Warning: Removed 129366 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-43.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-44.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>  cat(&quot;### for labels&quot;) </code></pre>
</div>
<div id="for-labels" class="section level3">
<h3>for labels</h3>
<pre class="r"><code>   plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC1_label),]
  plot_data &lt;- plot_data[,grep(&quot;control&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-45.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>   plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC2_label),]
  plot_data &lt;- plot_data[,grep(&quot;control&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-46.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code> plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC3_label),]
  plot_data &lt;- plot_data[,grep(&quot;control&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-47.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code> plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC4_label),]
  plot_data &lt;- plot_data[,grep(&quot;control&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-8-48.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>   cat(&#39;\n&#39;)</code></pre>
</div>
</div>
<div id="lmc-methylation-of-markers" class="section level2 tabset tabset-fade tabset-pills">
<h2>LMC methylation of markers</h2>
<pre class="r"><code>for (marks in colnames(datasets[[1]])[grep(&quot;_markers&quot;,colnames(datasets[[1]]) )]){
  
   cat(&#39;\n&#39;)
  
  cat(&quot;### Markers for &quot;,
      gsub(&quot;_markers&quot;, &quot;&quot;, marks), &quot; cells. &quot;,
      &quot;\n&quot;)

try(pheatmap(datasets[[1]][!is.na(datasets[[1]][,marks]),grep(&quot;gini&quot;, colnames(datasets[[1]]))], show_rownames = F, main = marks))
cat(&#39;\n&#39;)
  
}</code></pre>
<pre><code>
### Markers for  AT1  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  AT2  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  basal  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  ciliated  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  club  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  goblet  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  Krt8  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  AT1_AT2  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-9-8.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="all-samples" class="section level1">
<h1>All samples</h1>
<pre class="r"><code>datasets &lt;- datasets_combined[[1]]
T_all &lt;- MeDeCom:::getT.gini(D=as.matrix(datasets[[1]][,grep(&quot;Ms&quot;, colnames(datasets[[1]]))]), A = datasets[[2]], lambda = lambda_sel)
colnames(T_all) &lt;- paste0(&quot;gini_LMC&quot;, 1:K_sel)
datasets[[1]] &lt;- cbind(datasets[[1]], T_all)

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  &quot;[&lt;-&quot;(paste0(&quot;LMC&quot;, 1:4), value = NA_real_) %&gt;%
   &quot;[&lt;-&quot;(paste0(&quot;LMC&quot;, 1:4, &quot;_p&quot;), value = NA_real_)


for (LMCs in paste0(&quot;LMC&quot;, 1:4)){
  
est &lt;-lapply(1:nrow(datasets[[1]]), function(x)
  cor.test(datasets[[2]][LMCs,],  as.numeric(datasets[[1]][x,-c(1:5, which(colnames(datasets[[1]])==&quot;symbol&quot;), grep(&quot;LMC&quot;, colnames(datasets[[1]])))]), method = &quot;pears&quot;))
datasets[[1]][,LMCs] &lt;- unlist(lapply(est, function(x) x$estimate))
datasets[[1]][,paste0(LMCs, &quot;_p&quot;)] &lt;- -log10(unlist(lapply(est, function(x) x$p.value)))

}

#fun_filt &lt;- function(val, pval){
#  ifelse(abs(val)&gt;0.8 &amp; -log10(pval)&gt;5, symbol, NA)
#}

#datasets[[1]] &lt;- datasets[[1]] %&gt;%
#  mutate(across(c(Sepal.Length, Sepal.Width), round))
#    mutate(new_labels=replace(get(label), duplicated(get(label)), NA))
    
datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC1_label=ifelse(abs(LMC1)&gt;0.8 &amp; LMC1_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC2_label=ifelse(abs(LMC2)&gt;0.8 &amp; LMC2_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC3_label=ifelse(abs(LMC3)&gt;0.8 &amp; LMC3_p&gt;5, symbol, NA))

datasets[[1]] &lt;- datasets[[1]] %&gt;%
  mutate(LMC4_label=ifelse(abs(LMC4)&gt;0.8 &amp; LMC4_p&gt;5, symbol, NA))


vars &lt;- names(marker_lists_short)


datasets[[1]] &lt;- datasets[[1]] %&gt;%
#  mutate(across(colnames(.) %in% vars) ~ case_when(symbol %in% marker_lists_short[[.]]))
  mutate(AT1_markers=ifelse(symbol %in% marker_lists_short$`AT1 cells`, symbol, NA)) %&gt;% 
  mutate(AT2_markers=ifelse(symbol %in% marker_lists_short$`AT2 cells`, symbol, NA)) %&gt;%
  mutate(basal_markers=ifelse(symbol %in% marker_lists_short$`basal cells`, symbol, NA)) %&gt;%
  mutate(ciliated_markers=ifelse(symbol %in% marker_lists_short$`ciliated cells`, symbol, NA)) %&gt;%
  mutate(club_markers=ifelse(symbol %in% marker_lists_short$`club cells`, symbol, NA)) %&gt;%
  mutate(goblet_markers=ifelse(symbol %in% marker_lists_short$`goblet cells`, symbol, NA)) %&gt;%
  mutate(Krt8_markers=ifelse(symbol %in% marker_lists_short$`Krt8 ADI`, symbol, NA)) %&gt;%
  mutate(AT1_AT2_markers=ifelse(symbol %in% marker_lists_short$`Mixed AT1_AT2`, symbol, NA)) %&gt;%
  mutate(club_progenitor_markers=ifelse(symbol %in% marker_lists_short$`Club progenitor`, symbol, NA))


#volcano_plot &lt;- ggplot(datasets[[1]], aes(x=LMC1, y=-log10(LMC1_p), label=club_markers)) + 
#  geom_point()+geom_label_repel(max.overlaps = Inf)</code></pre>
<div id="correlation-of-promoter-methylation-with-lmcs-1" class="section level2 tabset tabset-fade tabset-pills">
<h2>Correlation of promoter methylation with LMCs</h2>
<pre class="r"><code>for (lmcs in paste0(&quot;LMC&quot;, 1:4)){
g &lt;- volcano_plot_correlation(data = datasets[[1]], x = lmcs, y = paste0(lmcs, &quot;_p&quot;), label = paste0(lmcs, &quot;_label&quot;))
print(g)
}</code></pre>
<pre><code>Warning: Removed 78529 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79552 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79425 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79531 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>for (marks in colnames(datasets[[1]])[grep(&quot;_markers&quot;,colnames(datasets[[1]]) )]){
  
   cat(&#39;\n&#39;)
  
  cat(&quot;### Markers for &quot;,
      gsub(&quot;_markers&quot;, &quot;&quot;, marks), &quot; cells. &quot;,
      &quot;\n&quot;)
  
  for (lmcs in paste0(&quot;LMC&quot;, 1:4)){
g &lt;- volcano_plot_correlation(data = datasets[[1]], x = lmcs, y = paste0(lmcs, &quot;_p&quot;), label = marks)
print(g)
  }
  

       cat(&#39;\n&#39;)
       
 plot_data &lt;- datasets[[1]][!is.na(datasets[[1]][,marks]),]
  plot_data &lt;- plot_data[,grep(&quot;Ms&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))

  cat(&#39;\n&#39;)
  
       
}</code></pre>
<div id="markers-for-at1-cells.-1" class="section level3">
<h3>Markers for AT1 cells.</h3>
<pre><code>Warning: Removed 79616 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79616 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79616 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79616 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-8.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-9.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-at2-cells.-1" class="section level3">
<h3>Markers for AT2 cells.</h3>
<pre><code>Warning: Removed 79618 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79618 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79618 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79618 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-13.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-14.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-basal-cells.-1" class="section level3">
<h3>Markers for basal cells.</h3>
<pre><code>Warning: Removed 79605 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-15.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79605 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-16.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79605 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-17.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79605 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-18.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-19.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-ciliated-cells.-1" class="section level3">
<h3>Markers for ciliated cells.</h3>
<pre><code>Warning: Removed 79613 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-20.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79613 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-21.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79613 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-22.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79613 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-23.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-24.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-club-cells.-1" class="section level3">
<h3>Markers for club cells.</h3>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-25.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-26.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-27.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-28.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-29.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-goblet-cells.-1" class="section level3">
<h3>Markers for goblet cells.</h3>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-30.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-31.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-32.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79610 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-33.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-34.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-krt8-cells.-1" class="section level3">
<h3>Markers for Krt8 cells.</h3>
<pre><code>Warning: Removed 79611 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-35.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79611 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-36.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79611 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-37.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79611 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-38.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-39.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-at1_at2-cells.-1" class="section level3">
<h3>Markers for AT1_AT2 cells.</h3>
<pre><code>Warning: Removed 79622 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-40.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79622 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-41.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79622 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-42.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79622 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-43.png" width="672" style="display: block; margin: auto;" /> <img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-44.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="markers-for-club_progenitor-cells." class="section level3">
<h3>Markers for club_progenitor cells.</h3>
<pre><code>Warning: Removed 79628 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-45.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79628 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-46.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79628 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-47.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning: Removed 79628 rows containing missing values (geom_label_repel).</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-48.png" width="672" style="display: block; margin: auto;" /> Error in hclust(d, method = method) : must have n &gt;= 2 objects to cluster</p>
<pre class="r"><code>  cat(&quot;### for labels&quot;) </code></pre>
</div>
<div id="for-labels-1" class="section level3">
<h3>for labels</h3>
<pre class="r"><code>   plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC1_label),]
  plot_data &lt;- plot_data[,grep(&quot;Ms&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-49.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>    plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC2_label),]
  plot_data &lt;- plot_data[,grep(&quot;Ms&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-50.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code> plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC3_label),]
  plot_data &lt;- plot_data[,grep(&quot;Ms&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-51.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code> plot_data &lt;- datasets[[1]][!is.na(datasets[[1]]$LMC4_label),]
  plot_data &lt;- plot_data[,grep(&quot;Ms&quot;, colnames(plot_data))]
try(pheatmap(plot_data, show_rownames = F))</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-11-52.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>   cat(&#39;\n&#39;)</code></pre>
</div>
</div>
<div id="lmc-methylation-of-markers-1" class="section level2 tabset tabset-fade tabset-pills">
<h2>LMC methylation of markers</h2>
<pre class="r"><code>for (marks in colnames(datasets[[1]])[grep(&quot;_markers&quot;,colnames(datasets[[1]]) )]){
  
   cat(&#39;\n&#39;)
  
  cat(&quot;### Markers for &quot;,
      gsub(&quot;_markers&quot;, &quot;&quot;, marks), &quot; cells. &quot;,
      &quot;\n&quot;)

try(pheatmap(datasets[[1]][!is.na(datasets[[1]][,marks]),grep(&quot;gini&quot;, colnames(datasets[[1]]))], show_rownames = F, main = marks))
cat(&#39;\n&#39;)
  
}</code></pre>
<pre><code>
### Markers for  AT1  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  AT2  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  basal  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  ciliated  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  club  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  goblet  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  Krt8  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  AT1_AT2  cells.  </code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-12-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>

### Markers for  club_progenitor  cells.  
Error in hclust(d, method = method) : must have n &gt;= 2 objects to cluster</code></pre>
</div>
</div>
<div id="gsea-like-plots" class="section level1">
<h1>GSEA like plots</h1>
<div id="epithelial-cell-types" class="section level2">
<h2>Epithelial cell types</h2>
<pre class="r"><code>mypal = pal_igv()(27)
df &lt;- readRDS(file=file.path(DATA, &quot;medecom/control_correlation_LMCs_no_Hopx.Rmd&quot;))

gsea_rank &lt;-  df %&gt;%
  arrange(desc(abs(LMC1)),desc(LMC1_p)) %&gt;%
  mutate(symbol=replace(symbol, duplicated(symbol), NA))%&gt;%
  dplyr::filter(!is.na(symbol)) %&gt;%
  arrange(LMC1)


lmc1_rank &lt;- gsea_rank$LMC1
names(lmc1_rank) &lt;- gsea_rank$symbol


fgseaRes &lt;- fgsea(pathways = marker_lists_short,
                  stats    = lmc1_rank,
                 minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = marker_lists_short, stats = lmc1_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.18% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(4,5,7,8), color = c(&quot;green&quot;, &quot;red&quot;, &quot;blue&quot;, &quot;black&quot;),  base_size = 11, rel_heights = c(1.5, 0.5, 1), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC1 correlation&quot;, ranks = lmc1_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gsea_rank &lt;-  df %&gt;%
  arrange(desc(abs(LMC4)),desc(LMC4_p)) %&gt;%
  mutate(symbol=replace(symbol, duplicated(symbol), NA))%&gt;%
  dplyr::filter(!is.na(symbol)) %&gt;%
  arrange(LMC4)


lmc4_rank &lt;- gsea_rank$LMC4
names(lmc4_rank) &lt;- gsea_rank$symbol

fgseaRes &lt;- fgsea(pathways = marker_lists_short,
                  stats    = lmc4_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = marker_lists_short, stats = lmc4_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.15% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(4,5,7,8), color = c(&quot;green&quot;, &quot;red&quot;, &quot;blue&quot;, &quot;black&quot;),  base_size = 11, rel_heights = c(1.5, 0.5, 1), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC4 correlation&quot;, ranks = lmc4_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gsea_rank &lt;-  df %&gt;%
  arrange(desc(abs(LMC3)),desc(LMC3_p)) %&gt;%
  mutate(symbol=replace(symbol, duplicated(symbol), NA))%&gt;%
  dplyr::filter(!is.na(symbol)) %&gt;%
  arrange(LMC3)


lmc3_rank &lt;- gsea_rank$LMC3
names(lmc3_rank) &lt;- gsea_rank$symbol

fgseaRes &lt;- fgsea(pathways = marker_lists_short,
                  stats    = lmc3_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = marker_lists_short, stats = lmc3_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.21% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(4,5,7,8), color = c(&quot;green&quot;, &quot;red&quot;, &quot;blue&quot;, &quot;black&quot;),  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gseaplot2(geneSetID = c(1:10), color = mypal[1:10],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gseaplot2(geneSetID = c(11:20), color = mypal[11:20],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gseaplot2(geneSetID = c(21:27), color = mypal[21:27],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
          title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = marker_lists_short, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-13-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>selected_markers_MCA &lt;- read.delim(&quot;V:/Reka/33_CoO_lung/CoO_Lung_Cancer/data/selected_markers_MCA.txt&quot;)
selected_markers_MCA  &lt;- apply(selected_markers_MCA, 2, function(x) mapIds(org.Mm.eg.db,
keys = x,
column = &quot;SYMBOL&quot;,
keytype = &quot;ENSEMBL&quot;))</code></pre>
<pre><code>&#39;select()&#39; returned 1:1 mapping between keys and columns
&#39;select()&#39; returned 1:1 mapping between keys and columns</code></pre>
<pre><code>&#39;select()&#39; returned 1:many mapping between keys and columns</code></pre>
<pre><code>&#39;select()&#39; returned 1:1 mapping between keys and columns</code></pre>
<pre><code>&#39;select()&#39; returned 1:many mapping between keys and columns</code></pre>
<pre class="r"><code>selected_markers_MCA_l &lt;- lapply(1:ncol(selected_markers_MCA), function(i) selected_markers_MCA[,i])
names(selected_markers_MCA_l) &lt;- colnames(selected_markers_MCA)

fgseaRes &lt;- fgsea(pathways = selected_markers_MCA_l,
                  stats    = lmc1_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = selected_markers_MCA_l, stats = lmc1_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.18% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(1:5), color = mypal[1:5],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
title = &quot;LMC1 correlation&quot;, ranks = lmc1_rank, geneSet = selected_markers_MCA_l, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fgseaRes &lt;- fgsea(pathways = selected_markers_MCA_l,
                  stats    = lmc4_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = selected_markers_MCA_l, stats = lmc4_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.15% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(1:5), color = mypal[1:5],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
title = &quot;LMC4 correlation&quot;, ranks = lmc4_rank, geneSet = selected_markers_MCA_l, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-14-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>fgseaRes &lt;- fgsea(pathways = selected_markers_MCA_l,
                  stats    = lmc3_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = selected_markers_MCA_l, stats = lmc3_rank,
minSize = 3, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.21% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(1:5), color = mypal[1:5],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = selected_markers_MCA_l, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-14-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>lung_cell_atlas_other_markers &lt;- read.delim(&quot;V:/Reka/33_CoO_lung/CoO_Lung_Cancer/data/lung_cell_atlas_other_markers.txt&quot;)
new_list &lt;- lapply(1:ncol(lung_cell_atlas_other_markers), function(x)  CapStr(tolower(lung_cell_atlas_other_markers[,x])))
new_list &lt;- lapply(new_list, function(x) x[1:28])
new_list &lt;- lapply(new_list, function(x) x[x %in% gsea_rank$symbol])
names(new_list) &lt;- colnames(lung_cell_atlas_other_markers)

fgseaRes &lt;- fgsea(pathways = new_list,
                  stats    = lmc3_rank,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)</code></pre>
<pre><code>Warning in fgsea(pathways = new_list, stats = lmc3_rank, minSize = 3,
maxSize = 500, : You are trying to run fgseaSimple. It is recommended to use
fgseaMultilevel. To run fgseaMultilevel, you need to remove the nperm argument
in the fgsea function call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.21% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre class="r"><code>gseaplot2(geneSetID = c(1:13), color = mypal[1:13],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = new_list, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gseaplot2(geneSetID = c(3:5), color = mypal[1:3],  base_size = 11, rel_heights = c(1.5, 0.2, 0.5), subplots = 1:3,  pvalue_table = FALSE, ES_geom = &quot;line&quot;,
                     title = &quot;LMC3 correlation&quot;, ranks = lmc3_rank, geneSet = new_list, pvaluetable = fgseaRes)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-15-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Heatmap all pathways combined


marker_lists_short &lt;- marker_lists_short[-19]
all_pw &lt;- c(marker_lists_short, selected_markers_MCA_l, new_list)

for (lmcs in list(lmc1_rank, lmc3_rank, lmc4_rank)){

gsdata2 &lt;- do.call(rbind, lapply(1:length(all_pw), function(id) gsInfo2(geneSetID=id, geneset = all_pw, res = lmcs)))

fgseaRes &lt;- fgsea(pathways = all_pw,
                  stats    = lmcs,
                  minSize  = 3,
                  maxSize  = 500, nperm=1000)

gsdata3 &lt;- gsdata2 %&gt;% 
  inner_join(fgseaRes[,c(&quot;pathway&quot;, &quot;pval&quot;)], by= c(&quot;Description&quot;=&quot;pathway&quot;)) %&gt;% 
  mutate(significant=ifelse(pval&lt;0.05, &quot; *&quot;, NA)) %&gt;%
  mutate(Description=factor(Description, levels = fgseaRes$pathway[order(fgseaRes$ES, decreasing = T)]))
  
p &lt;- ggplot(gsdata3)+geom_tile(aes(Description, factor(x), fill=runningScore)) +scale_fill_gradient2(low=&quot;darkred&quot;, mid=&quot;white&quot;, high=&quot;darkblue&quot;)+theme(axis.text.x = element_blank(), axis.title.x = element_blank(), axis.ticks.x = element_blank(), axis.text.y=element_text())+ coord_flip()
print(p)
}</code></pre>
<pre><code>Warning in fgsea(pathways = all_pw, stats = lmcs, minSize = 3, maxSize = 500, :
You are trying to run fgseaSimple. It is recommended to use fgseaMultilevel. To
run fgseaMultilevel, you need to remove the nperm argument in the fgsea function
call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.18% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<pre><code>Warning in fgsea(pathways = all_pw, stats = lmcs, minSize = 3, maxSize = 500, :
You are trying to run fgseaSimple. It is recommended to use fgseaMultilevel. To
run fgseaMultilevel, you need to remove the nperm argument in the fgsea function
call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.21% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Warning in fgsea(pathways = all_pw, stats = lmcs, minSize = 3, maxSize = 500, :
You are trying to run fgseaSimple. It is recommended to use fgseaMultilevel. To
run fgseaMultilevel, you need to remove the nperm argument in the fgsea function
call.</code></pre>
<pre><code>Warning in preparePathwaysAndStats(pathways, stats, minSize, maxSize, gseaParam, : There are ties in the preranked stats (0.15% of the list).
The order of those tied genes will be arbitrary, which may produce unexpected results.</code></pre>
<p><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-16-2.png" width="672" style="display: block; margin: auto;" /><img src="figure/MeDeCom_results_noHopx.Rmd/unnamed-chunk-16-3.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] TxDb.Mmusculus.UCSC.mm10.knownGene_3.10.0
 [2] GenomicFeatures_1.42.0                   
 [3] cerebroApp_1.2.2                         
 [4] harmony_1.0                              
 [5] Rcpp_1.0.5                               
 [6] SingleR_1.4.0                            
 [7] patchwork_1.1.0                          
 [8] cowplot_1.1.0                            
 [9] Seurat_3.2.2                             
[10] ggsci_2.9                                
[11] fgsea_1.16.0                             
[12] enrichplot_1.10.0                        
[13] DOSE_3.16.0                              
[14] ggrepel_0.9.0.9999                       
[15] dplyr_1.0.2                              
[16] readxl_1.3.1                             
[17] annotatr_1.16.0                          
[18] pheatmap_1.0.12                          
[19] kableExtra_1.3.1                         
[20] plotly_4.9.2.1                           
[21] ggpubr_0.4.0                             
[22] ggplot2_3.3.3                            
[23] homerkit_0.1.0                           
[24] org.Mm.eg.db_3.12.0                      
[25] AnnotationDbi_1.52.0                     
[26] HDF5Array_1.18.0                         
[27] rhdf5_2.34.0                             
[28] DelayedArray_0.16.0                      
[29] Matrix_1.2-18                            
[30] rtracklayer_1.49.5                       
[31] ChIPseeker_1.26.0                        
[32] doParallel_1.0.16                        
[33] iterators_1.0.13                         
[34] foreach_1.5.1                            
[35] DSS_2.38.0                               
[36] bsseq_1.26.0                             
[37] SummarizedExperiment_1.20.0              
[38] MatrixGenerics_1.2.0                     
[39] matrixStats_0.57.0                       
[40] GenomicRanges_1.42.0                     
[41] GenomeInfoDb_1.26.0                      
[42] IRanges_2.24.0                           
[43] S4Vectors_0.28.0                         
[44] BiocParallel_1.24.1                      
[45] Biobase_2.50.0                           
[46] BiocGenerics_0.36.0                      
[47] workflowr_1.6.2.9000                     

loaded via a namespace (and not attached):
  [1] rsvd_1.0.3                             
  [2] ica_1.0-2                              
  [3] Rsamtools_2.6.0                        
  [4] lmtest_0.9-38                          
  [5] rprojroot_2.0.2                        
  [6] crayon_1.3.4                           
  [7] MASS_7.3-53                            
  [8] rhdf5filters_1.2.0                     
  [9] nlme_3.1-149                           
 [10] backports_1.2.0                        
 [11] colourpicker_1.1.0                     
 [12] GOSemSim_2.16.1                        
 [13] rlang_0.4.10                           
 [14] XVector_0.30.0                         
 [15] ROCR_1.0-11                            
 [16] irlba_2.3.3                            
 [17] minfi_1.36.0                           
 [18] limma_3.46.0                           
 [19] rjson_0.2.20                           
 [20] bit64_4.0.5                            
 [21] glue_1.4.2                             
 [22] rngtools_1.5                           
 [23] sctransform_0.3.1                      
 [24] methylumi_2.36.0                       
 [25] regioneR_1.22.0                        
 [26] dotCall64_1.0-0                        
 [27] shinydashboard_0.7.1                   
 [28] haven_2.3.1                            
 [29] tidyselect_1.1.0                       
 [30] rio_0.5.16                             
 [31] fitdistrplus_1.1-1                     
 [32] XML_3.99-0.5                           
 [33] tidyr_1.1.2                            
 [34] zoo_1.8-8                              
 [35] GenomicAlignments_1.26.0               
 [36] xtable_1.8-4                           
 [37] formattable_0.2.0.1                    
 [38] magrittr_2.0.1                         
 [39] evaluate_0.14                          
 [40] zlibbioc_1.36.0                        
 [41] doRNG_1.8.2                            
 [42] rstudioapi_0.13                        
 [43] miniUI_0.1.1.1                         
 [44] whisker_0.4                            
 [45] rpart_4.1-15                           
 [46] fastmatch_1.1-0                        
 [47] treeio_1.14.0                          
 [48] maps_3.3.0                             
 [49] fields_11.6                            
 [50] shiny_1.5.0                            
 [51] GSVA_1.38.0                            
 [52] BiocSingular_1.6.0                     
 [53] xfun_0.19                              
 [54] askpass_1.1                            
 [55] multtest_2.46.0                        
 [56] cluster_2.1.0                          
 [57] caTools_1.18.0                         
 [58] tidygraph_1.2.0                        
 [59] tibble_3.0.4                           
 [60] interactiveDisplayBase_1.28.0          
 [61] base64_2.0                             
 [62] ff_4.0.4                               
 [63] scrime_1.3.5                           
 [64] ape_5.4-1                              
 [65] listenv_0.8.0                          
 [66] reshape_0.8.8                          
 [67] shinyWidgets_0.5.4                     
 [68] Biostrings_2.58.0                      
 [69] png_0.1-7                              
 [70] permute_0.9-5                          
 [71] future_1.20.1                          
 [72] withr_2.3.0                            
 [73] bitops_1.0-6                           
 [74] ggforce_0.3.2                          
 [75] plyr_1.8.6                             
 [76] cellranger_1.1.0                       
 [77] GSEABase_1.52.0                        
 [78] bumphunter_1.32.0                      
 [79] pillar_1.4.7                           
 [80] gplots_3.1.0                           
 [81] fs_1.5.0                               
 [82] DelayedMatrixStats_1.12.0              
 [83] vctrs_0.3.6                            
 [84] ellipsis_0.3.1                         
 [85] generics_0.1.0                         
 [86] tools_4.0.3                            
 [87] foreign_0.8-80                         
 [88] munsell_0.5.0                          
 [89] tweenr_1.0.1                           
 [90] fastmap_1.0.1                          
 [91] compiler_4.0.3                         
 [92] abind_1.4-5                            
 [93] httpuv_1.5.4                           
 [94] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [95] beanplot_1.2                           
 [96] GenomeInfoDbData_1.2.4                 
 [97] gridExtra_2.3                          
 [98] lattice_0.20-41                        
 [99] deldir_0.2-3                           
[100] later_1.1.0.1                          
[101] BiocFileCache_1.14.0                   
[102] jsonlite_1.7.2                         
[103] scales_1.1.1                           
[104] graph_1.68.0                           
[105] tidytree_0.3.3                         
[106] pbapply_1.4-3                          
[107] carData_3.0-4                          
[108] sparseMatrixStats_1.2.0                
[109] genefilter_1.72.0                      
[110] lazyeval_0.2.2                         
[111] promises_1.1.1                         
[112] spatstat_1.64-1                        
[113] car_3.0-10                             
[114] R.utils_2.10.1                         
[115] goftest_1.2-2                          
[116] spatstat.utils_1.17-0                  
[117] reticulate_1.18                        
[118] nor1mix_1.3-0                          
[119] rmarkdown_2.5                          
[120] openxlsx_4.2.3                         
[121] webshot_0.5.2                          
[122] siggenes_1.64.0                        
[123] Rtsne_0.15                             
[124] forcats_0.5.0                          
[125] BSgenome_1.58.0                        
[126] uwot_0.1.8                             
[127] igraph_1.2.6                           
[128] survival_3.2-7                         
[129] yaml_2.2.1                             
[130] plotrix_3.7-8                          
[131] htmltools_0.5.1.1                      
[132] memoise_1.1.0                          
[133] locfit_1.5-9.4                         
[134] quadprog_1.5-8                         
[135] graphlayouts_0.7.1                     
[136] viridisLite_0.3.0                      
[137] digest_0.6.27                          
[138] assertthat_0.2.1                       
[139] mime_0.9                               
[140] rappdirs_0.3.1                         
[141] spam_2.5-1                             
[142] RSQLite_2.2.1                          
[143] future.apply_1.6.0                     
[144] data.table_1.13.2                      
[145] blob_1.2.1                             
[146] R.oo_1.24.0                            
[147] preprocessCore_1.52.0                  
[148] labeling_0.4.2                         
[149] splines_4.0.3                          
[150] Rhdf5lib_1.12.0                        
[151] illuminaio_0.32.0                      
[152] AnnotationHub_2.22.0                   
[153] RCurl_1.98-1.2                         
[154] broom_0.7.2                            
[155] hms_0.5.3                              
[156] colorspace_2.0-0                       
[157] BiocManager_1.30.10                    
[158] aplot_0.0.6                            
[159] GEOquery_2.58.0                        
[160] mclust_5.4.6                           
[161] RANN_2.6.1                             
[162] parallelly_1.21.0                      
[163] R6_2.5.0                               
[164] grid_4.0.3                             
[165] ggridges_0.5.2                         
[166] lifecycle_0.2.0                        
[167] zip_2.1.1                              
[168] curl_4.3                               
[169] ggsignif_0.6.0                         
[170] leiden_0.3.5                           
[171] DO.db_2.9                              
[172] RnBeads_2.8.1                          
[173] qvalue_2.22.0                          
[174] RcppAnnoy_0.0.16                       
[175] RColorBrewer_1.1-2                     
[176] stringr_1.4.0                          
[177] htmlwidgets_1.5.2                      
[178] beachmat_2.6.1                         
[179] polyclip_1.10-0                        
[180] biomaRt_2.46.0                         
[181] purrr_0.3.4                            
[182] shadowtext_0.0.7                       
[183] rvest_0.3.6                            
[184] methrix_1.3.25                         
[185] mgcv_1.8-33                            
[186] globals_0.13.1                         
[187] openssl_1.4.3                          
[188] MeDeCom_1.0.0                          
[189] codetools_0.2-16                       
[190] GO.db_3.12.1                           
[191] gtools_3.8.2                           
[192] prettyunits_1.1.1                      
[193] dbplyr_2.0.0                           
[194] R.methodsS3_1.8.1                      
[195] gtable_0.3.0                           
[196] DBI_1.1.0                              
[197] git2r_0.27.1                           
[198] tensor_1.5                             
[199] httr_1.4.2                             
[200] highr_0.8                              
[201] KernSmooth_2.23-17                     
[202] stringi_1.5.3                          
[203] progress_1.2.2                         
[204] msigdbr_7.2.1                          
[205] reshape2_1.4.4                         
[206] farver_2.0.3                           
[207] annotate_1.68.0                        
[208] viridis_0.5.1                          
[209] ggtree_2.4.0                           
[210] DT_0.16                                
[211] xml2_1.3.2                             
[212] rvcheck_0.1.8                          
[213] boot_1.3-25                            
[214] BiocNeighbors_1.8.0                    
[215] readr_1.4.0                            
[216] BiocVersion_3.12.0                     
[217] bit_4.0.4                              
[218] scatterpie_0.1.5                       
[219] spatstat.data_1.4-3                    
[220] ggraph_2.0.3                           
[221] pkgconfig_2.0.3                        
[222] rstatix_0.6.0                          
[223] knitr_1.30                             </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
